builtin/providers/google/resource_compute_instance_group.go

```diff
@@ -0,0 +1,301 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceComputeInstanceGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeInstanceGroupCreate,
+		Read:   resourceComputeInstanceGroupRead,
+		Update: resourceComputeInstanceGroupUpdate,
+		Delete: resourceComputeInstanceGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"fingerprint": &schema.Schema{
```

Comment:
This isn't necessary to have in the schema. In `update`, you want to do a 3-way merge between the old TF state, the new TF state, and the current server state. When you retrieve that server state during the update you get the fingerprint you need for the update to succeed. Since this fingerprint is only valid and needed for that single update, it doesn't make sense to keep in the schema. I know that some resources like `google_compute_instance` keep fingerprint in the schema, but it's a mistake and a pain to remove.